be/src/olap/skiplist.h

```diff
@@ -186,7 +182,7 @@ struct SkipList<Key,Comparator>::Node {
 template<typename Key, class Comparator>
 typename SkipList<Key,Comparator>::Node*
 SkipList<Key,Comparator>::NewNode(const Key& key, int height) {
-    char* mem = arena_->AllocateAligned(
+    char* mem = (char*)_mem_pool->allocate(
```

Comment:
Is this OK if we allocate not aligned?

Reply:
If I don't think wrong, _mem_pool->allocate is equal with arena_->AllocateAligned.
```
    /// Allocates a section of memory of 'size' bytes with DEFAULT_ALIGNMENT at the end
    /// of the the current chunk. Creates a new chunk if there aren't any chunks
    /// with enough capacity.
    uint8_t* allocate(int64_t size) {
        return allocate<false>(size, DEFAULT_ALIGNMENT);
    }
```